src/pages/PhonePage/PhonePage.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from 'react';
+import { Product } from '../../types/Product';
+import { getAllProducts } from '../../helpers/fetchProducts';
+import { ProductType } from '../../types/ProductType';
+import { ProductPage } from '../ProductPage';
+
+import './PhonePage.scss';
+
+export const PhonePage = () => {
```

Comment:
PhonesPage - it renders the list of Phones not one item

The same below - ProductsPage